bpf/bpf_overlay.c

```diff
@@ -145,9 +146,13 @@ static __always_inline int handle_ipv6(struct __ctx_buff *ctx,
 
 	/* Deliver to local (non-host) endpoint: */
 	ep = lookup_ip6_endpoint(ip6);
-	if (ep && !(ep->flags & ENDPOINT_MASK_HOST_DELIVERY))
+	if (ep && !(ep->flags & ENDPOINT_MASK_HOST_DELIVERY)) {
+		ret = accept(ctx, ep->lxc_id);
+		if (IS_ERR(ret))
+			return ret;
```

Comment:
Looks great! Small nit: Could we move this directly into ipv{4,6}_local_delivery()?